node/service/res/crab/crab.json

```diff
@@ -3,8 +3,8 @@
 	"id": "crab",
 	"chainType": "Live",
 	"bootNodes": [
-		"/dns/g1.p2p.crab.darwinia.network/tcp/30333/p2p/12D3KooWFqHZkyv6iabxxqiHdNjWb4c7EfmBqMNCyqLCCVZm8yyQ",
-		"/dns/g2.p2p.crab.darwinia.network/tcp/30333/p2p/12D3KooWPiza2NAD6CjdBGtfUd3pfDnZXysYKzumejGHafW3Y8xP"
+		"/dns/g1.crab-p2p.darwinia.network/tcp/30333/p2p/12D3KooWFqHZkyv6iabxxqiHdNjWb4c7EfmBqMNCyqLCCVZm8yyQ",
+		"/dns/g2.crab-p2p.darwinia.network/tcp/30333/p2p/12D3KooWPiza2NAD6CjdBGtfUd3pfDnZXysYKzumejGHafW3Y8xP"
```

Comment:
Why update those addresses? The old node is no longer maintained?

Reply:
Old domain was dropped.